demo/talk_to_sambanova/app.py

```diff
@@ -76,6 +77,7 @@ def response(
     additional_outputs_handler=lambda *a: (a[2], a[3]),
     concurrency_limit=20 if get_space() else None,
     rtc_configuration=get_cloudflare_turn_credentials_async,
+    server_rtc_configuration=get_cloudflare_turn_credentials(ttl=36_000),
```

Comment:
nit: passing the same credentials to both `rtc_configuration` and `server_rtc_configuration` would be cleaner?
```python
rtc_configuration = get_cloudflare_turn_credentials(ttl=36_000)
stream = Stream(
    ReplyOnPause(
        response,  # type: ignore
        input_sample_rate=16000,
    ),
    mode="send",
    modality="audio",
    additional_inputs=[chatbot, state],
    additional_outputs=[chatbot, state],
    additional_outputs_handler=lambda *a: (a[2], a[3]),
    concurrency_limit=20 if get_space() else None,
    rtc_configuration=rtc_configuration,
    server_rtc_configuration=rtc_configuration,
)
```

Reply:
Yes but I would rather fetch short lived credentials in the client since they could be read by a user and used in a different application